src/GitReportCL/SetGitReport.cs

```diff
@@ -0,0 +1,32 @@
+﻿
+
+namespace GitReportCL
+{
+    using System;
+    using System.Globalization;
+    using System.Threading;
```

Comment:
Usually we place usings outside of `namespace`. Move usings outside of `namespace`.

This should be applied to every file in code base.